src/main/resources/assets/securitycraft/lang/en_gb.json

```diff
@@ -3,12 +3,31 @@
+        "block.securitycraft.reinforced_chiseled_stone_bricks": "Reinforced Chiselled Stone Bricks",
         "block.securitycraft.reinforced_gray_terracotta": "Reinforced Grey Terracotta",
         "block.securitycraft.reinforced_light_gray_terracotta": "Reinforced Light Grey Terracotta",
+        "block.securitycraft.reinforced_gold_block": "Reinforced Gold Block",
+        "block.securitycraft.reinforced_iron_block": "Reinforced Iron Block",
+        "block.securitycraft.reinforced_diamond_block": "Reinforced Diamond Block",
+        "block.securitycraft.reinforced_emerald_block": "Reinforced Emerald Block",
+        "block.securitycraft.reinforced_coal_block": "Reinforced Coal Block",
         "block.securitycraft.reinforced_gray_wool": "Reinforced Grey Wool",
         "block.securitycraft.reinforced_light_gray_wool": "Reinforced Light Grey Wool",
+        "block.securitycraft.reinforced_quartz_block": "Reinforced Quartz Block",
+        "block.securitycraft.reinforced_chiseled_quartz_block": "Reinforced Chiselled Quartz Block",
+        "block.securitycraft.reinforced_chiseled_red_sandstone": "Reinforced Chiselled Red Sandstone",
         "block.securitycraft.reinforced_gray_concrete": "Reinforced Grey Concrete",
         "block.securitycraft.reinforced_light_gray_concrete": "Reinforced Light Grey Concrete",
         "block.securitycraft.reinforced_gray_carpet": "Reinforced Grey Carpet",
-        "block.securitycraft.reinforced_light_gray_carpet": "Reinforced Light Grey Carpet"
+        "block.securitycraft.reinforced_light_gray_carpet": "Reinforced Light Grey Carpet",
+        "block.securitycraft.crystal_quartz": "Crystal Quartz Block",
+        "block.securitycraft.chiseled_crystal_quartz": "Chiselled Crystal Quartz Block",
+        "block.securitycraft.reinforced_crystal_quartz_block": "Reinforced Crystal Quartz Block",
+        "block.securitycraft.reinforced_chiseled_crystal_quartz_block": "Reinforced Chiselled Crystal Quartz Block",
+        "block.securitycraft.reinforced_redstone_block": "Reinforced Redstone Block",
+        
+        "help.securitycraft.block_pocket_manager.info": "The Block Pocket Manager is used to manage the Block Pocket, which is a room that only its owner and whitelisted players can enter. When opening its GUI, you see a selection of sizes. Choose one, then build the Block Pocket around the manager. Make sure that the manager is placed on the outside of the bottom layer. The Block Pocket is a cube of the size you chose. The corners are built with Reinforced Chiselled Crystal Quartz, the blocks between the corners are built with correctly oriented Reinforced Crystal Quartz Pillars, and the floor/wall/ceiling is built with Block Pocket Walls. See this image for an example of how it looks: https://i.imgur.com/opc7dqO.png - Clicking \"Activate\" after the structure is built will activate any added modules. See the customization GUI for more information (rightclick with Universal Block Modifier)."
+
+
```

Comment:
Missed these two empty lines in my last review, sorry about that. Once they're removed, I'll merge!

Reply:
I have absolutely no clue how the empty lines came there, but now they're gone.